arraycontainer.go

```diff
@@ -439,6 +439,7 @@ func (ac *arrayContainer) lazyorArray(value2 *arrayContainer) container {
 }
 
 func (ac *arrayContainer) and(a container) container {
+	//p("ac.and() called")
```

Comment:
Actually I meant to discuss this when the opportunity arose.

Since print based debugging is the reality for all Go programmers, I actually prefer to leave the commented out prints in place. Why make someone go through the hassle of carefully retyping all that again just to debug something the next time. Often that person is me! The comments don't hurt anything, and they do make it much easier to track down problems.